### PR TITLE
change translation of WETDAYS and WETDAYS_PROP

### LINKED ISSUE #1825: Change translation of WETDAYS
### Addressing a Problem?

I think we can do better than "jours mouillés".

### Potential Solution

_No response_

### Additional context

_No response_

### Contribution

- [X] I would be willing/able to open a Pull Request to contribute this feature.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by huard**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #1825 
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] CHANGELOG.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Updates the French translation for `wet days` and `wet days prop`

### Does this PR introduce a breaking change?

No

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+v0.52.0 (unreleased)
+--------------------
+Contributors to this version: Trevor James Smith (:user:`Zeitsperre`), David Huard (:user:`huard`).
```
Comment (Zeitsperre) [inline CHANGELOG.rst:7]:
```suggestion
Contributors to this version: David Huard (:user:`huard`).
```

File: xclim/data/fr.json [hunk 2]
```diff
@@ -204,16 +204,16 @@
...
   },
   "WETDAYS": {
-    "long_name": "Nombre de jours où la précipitation est au-dessus ou égale à {thresh}",
-    "description": "Nombre {freq:m} de jours où la précipitation est au-dessus ou égale à {thresh}.",
-    "title": "Jours mouillés",
-    "abstract": "Nombre de jours où la précipitation est au-dessus ou égale à un seuil donné."
+    "long_name": "Nombre de jours où la précipitation est supérieure ou égale à {thresh}",
+    "description": "Nombre {freq:m} de jours où la précipitation est supérieure ou égale à {thresh}.",
+    "title": "Jours pluvieux",
+    "abstract": "Nombre de jours où la précipitation est supérieure ou égale à un seuil donné."
```
Comment (Zeitsperre) [inline xclim/data/fr.json:210]: Un des idées que nous avons discutées, c'était l'idée d'intégrer `{op}` dans plus des indicateurs et de passer `{op}` vers les descriptions. On a commencé avec les degrés jours de croissance. Qu'en penses-tu ?

File: CHANGELOG.rst (hunk 1 above)
Comment (Zeitsperre) [reply CHANGELOG.rst:7]: Early bird gets the contributor credit.

**Review by Zeitsperre — APPROVED** [review]

File: xclim/data/fr.json (hunk 2 above)
Comment (huard) [reply xclim/data/fr.json:210]: D'accord, mais peut-être mieux de faire ça dans un autre PR ?

**Pull request merged**